tools/mtouch/mtouch.cs

```diff
@@ -1570,7 +1570,7 @@ public static void CreateDsym (string output_dir, string appname, string dsym_di
 		{
 			string quoted_app_path = StringUtils.Quote (Path.Combine (output_dir, appname));
 			string quoted_dsym_dir = StringUtils.Quote (dsym_dir);
-			RunDsymUtil (string.Format ("{0} -t 4 -z -o {1}", quoted_app_path, quoted_dsym_dir));
+			RunDsymUtil (string.Format ("{0} -num-threads 4 -z -o {1}", quoted_app_path, quoted_dsym_dir));
```

Comment:
same here, `-z` seems problematic

Reply:
`-z` switch does show for me: `xcrun dsymutil -help ` last documented switch option

```
-z                      Alias for --minimize
```